mantidproject/mantid
# Simplify docs configuration

Description of work.

At the dev meeting we noticed that the `matplotlib.sphinxext.plot_directive` did not work correctly despite working in all other tested projects. It was discovered that this was due to our setup where the config file is outside of the source tree.

These changes move to a setup of the `docs` folder that is more inline with standard Sphinx setups. 

**To test:**

* Enable HTML docs build target and build `docs-html`. Check everything looks okay.
* Build the `docs-qthelp` target and ensure that this works as expected.
* Add a temporary page with an embedded matplotlib figure (https://matplotlib.org/sampledoc/extensions.html#inserting-matplotlib-plots) and check that building this gives a plot with functioning links.

*No issue*

**Release Notes** 

*Does not need to be in the release notes as it is a behind the scenes change*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
